src/Valuestore.php

```diff
@@ -235,32 +235,34 @@ public function pull(string $name): array|string|float|int|null
 
     /**
      * Increment a value from the store.
-     *
-     * @param string $name
-     * @param int    $by
-     *
-     * @return string|int|null
```

Comment:
Here I wasn't really sure about the intention. To me it's too much magic to allow incrementing strings.